homeassistant/components/gpsd/const.py

```diff
@@ -0,0 +1,12 @@
+"""Constants for the GPSD integration."""
+
+from typing import Final
+
+DOMAIN: Final = "gpsd"
+
+ATTR_CLIMB: Final = "climb"
+ATTR_SPEED: Final = "speed"
+
+DEFAULT_HOST: Final = "localhost"
+DEFAULT_NAME: Final = "GPS"
+DEFAULT_PORT: Final = 2947
```

Comment:
Are these reused? If not it is okay to leave them in the platform code,

Reply:
You mean the DEFAULT_* ones? Those are not, the others will be, I'm planning to add a coordinator to GPSD. Should I move back the DEFAULT_* ones?

Reply:
I suggest you add the move of those constants in the coordinator PR. But leave constants as long as they are only used withing the module.